boot/storage.txt

```diff
@@ -0,0 +1,485 @@
+    and features exist for them, refer to the Arch Wiki [9].
+
+
+    [3.2] Swap Partition
+    ____________________________________________________________________________ 
+    
+    To set up a partition as a Linux swap space, the mkswap command is use. 
+    Replace X with the drive where the swap partition is located, and Y by the
+    partition on that drive that will be formatted as swap.
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   mkswap /dev/sdXY                                                       |
+    |   swapon /dev/sdXY                                                       |
+    |                                                                          |
+    +--------------------------------------------------------------------------+  
+
+
+    [3.3] Swap file
+    ____________________________________________________________________________    
+    As an alternative to creating an entire partition, a swap file offers 
+    similar functionality, and the added benefits of being able to vary its size 
+    on-the-fly, as well as being more easily removed. This may be especially 
+    desirable if disk space is at a premium.
+
+    The most straightforward way to create a swap file is to use dd:
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ dd if=/dev/zero of=$LOCATION bs=$BYTES count=$SIZE                   |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    $LOCATION is the partition where the swapfile should be located. For the 
+    next few commands, assume $LOCATION=/swapfile.
+
+    $BYTES is The block size, which is determined by the filesystem on the 
+    partition where $LOCATION is. This setting is important for optimizing
+    read/write throughput. dd defaults to 512 bytes, but this is suboptimal for 
+    drives with larger supported block sizes. To determine the supported block
+    size for your device, run dumpe2fs (provided by e2fsprogs):
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ dumpe2fs -h /dev/sdXY | grep 'Block size'                            |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    For instance, if the result is 4096, bs=4k should be chosen. 
+    
+    $SIZE is the space you wish to allocate to the swapfile, suffixed by
+    kilobytes (kB), kibibytes (k), megabytes (M), mebibytes (MB), etc.
```

Comment:
Make all units uppercase? Are they case-sensitive in the first place? If not, it's probably nice to use consistent casing here.

Reply:
They are case sensitize. `dd` is a dumb and broken command.
What other program differentiates between megabytes and mebibytes ffs